tests/testthat/test-bound-prediction.R

```diff
@@ -0,0 +1,55 @@
+test_that("lower_limit bounds for numeric predictions", {
+  skip_if_not_installed("modeldata")
+  library(dplyr)
+  library(rlang)
+  data("solubility_test", package = "modeldata")
+  tune2 <- function() call("tune", "test")
```

Comment:
Is this function ever going to have to deal with `tune()` values. I guess I'm using my mental model from parsnip here, where upstream packages take care of `tune()` calls before we send specs off to `fit()`.

Reply:
It might erroneously receive that from users. I'd err on the side of over-checking

Reply:
I added an error for these values (instead of just passing)